sdk/src/clearingHouse.ts

```diff
@@ -925,6 +925,49 @@ export class ClearingHouse {
 		return result;
 	}
 
+	private async getTokenAccountCreationIxs(
+		tokenMintAddress: PublicKey
+	): Promise<{
+		ixs: anchor.web3.TransactionInstruction[];
+		signers: Signer[];
+		pubkey: PublicKey;
+	}> {
+		const accountKeypair = new Keypair();
+
+		const result = {
+			ixs: [],
+			signers: [],
+			pubkey: accountKeypair.publicKey,
+		};
+
+		const rentSpaceLamports = new BN(LAMPORTS_PER_SOL / 100);
+
+		const authority = this.wallet.publicKey;
+
+		result.ixs.push(
+			SystemProgram.createAccount({
+				fromPubkey: authority,
+				newAccountPubkey: accountKeypair.publicKey,
+				lamports: rentSpaceLamports.toNumber(),
+				space: 165,
+				programId: TOKEN_PROGRAM_ID,
+			})
+		);
+
+		result.ixs.push(
+			Token.createInitAccountInstruction(
```

Comment:
going to want to do `createAssociatedTokenAccountInstruction`, see tokenFaucet.createAssociatedTokenAccountAndMintToInstructions